test/python/expected/upgrade_validation/expected_dependency.out

```diff
@@ -612,6 +612,7 @@ Function sys.suser_name()
 Function sys.suser_name_internal(oid)
 Function sys.suser_sid()
 Function sys.suser_sname()
+Function sys.switchoffset(text,text)
```

Comment:
Add dependency test for this function call.

Reply:
Added